tests/test_fingerprints.py

```diff
@@ -387,3 +388,160 @@ def test_splif_similarity():
                                    0.727, 0.772, 0.747, 0.585, 0.535,
                                    0.681, 0.554, 0.736, 0.729])
     assert_array_almost_equal(outcome, target_outcome, decimal=3)
+
+
+def test_plecfp():
```

Comment:
We need to test **shapes** of FP based on `close_contacts` number for non-default `depth_ligand` and `depth_protein`, in particular test if it works as intended for `depth_ligand > depth_protein`.